apps/dashboard/src/app/activity/activity-download.component.html

```diff
@@ -29,24 +29,39 @@ <h1>{{ 'navbar.activity' | translate }}</h1>
     </pa-tabs>
     <div class="download-list">
       @for (item of activity[selectedActivityTab] | async; track item.month) {
```

Comment:
I think I wrote this line wrong.
Could we change it to `; track selectedActivityTab + item.month` ?